### DISCUSSION TITLE
What if it is possible to make it read mode only?

### DISCUSSION THREAD

**Discussion by beinto** [Foblex/f-flow#72] [Feature Request]
It would be great if we could make the canvas read only mode. Is it possible to disable all iteractions on the model?

**Comment by siarheihuzarevich** [comment]
@beinto Hello, yes you can do it. Every directive and component has a disabled attribute. It disables drag and drop functionality for that component. Also you can set this attribute for fDraggable directive and it will disable drag and drop for the whole flow. Please, be careful when reading the documentation

**Comment by beinto** [comment]
thanks
